bundle/Resources/config/view.yml

```diff
@@ -45,5 +42,5 @@ services:
             - 'eZ\Publish\Core\MVC\Symfony\Matcher\ContentBased'
         calls:
             - [setContainer, ["@service_container"]]
-            - [setMatchConfig, [$content_view_defaults$]]
+            - [setMatchConfig, [$ng_content_view_defaults$]]
```

Comment:
Maybe call this `ngcontent_view_defaults` to keep it consistent with `ngcontent_view` rules?

Reply:
Updated in https://github.com/netgen/ezplatform-site-api/pull/130/commits/c6c095ea93bd4741180f08d7e925bb1f939e9f57, I guess we are now stuck with it.